rakelib/plugin.rake

```diff
@@ -29,6 +32,32 @@ namespace "plugin" do
     LogStash::PluginManager::Main.run("bin/logstash-plugin", ["remove", plugin] + more_plugins)
   end
 
+  def list_plugins(search=nil, expand: true, verbose: false)
+    require_relative "../lib/pluginmanager/main"
+    args = []
+    args << "--verbose" if verbose
+    # args << (expand ? "--expand" : "--no-expand")
```

Comment:
Probably wont need `--expand` here?

Reply:
Looks like the caller sends along `expand: false` but then its ignored here. I think the `    stdout.lines.map(&:chomp).select { |p| p.start_with?('logstash-') }` line cleans the plubin names anyway?